### DISCUSSION TITLE
Getting build error "no such file or directory" for ~/...DerivedData... Debug/PackageFrameworks/SwiftUIIntrospect-Dynamic.framework/Versions/A/SwiftUIIntrospect-Dynamic'"

### DISCUSSION THREAD

**Discussion by DuncanMC** [siteline/SwiftUI-Introspect#391] [Q&A]
I'm using Xcode 15.0, and trying to add swiftui-introspect to my multiplatfrom SwiftUI project.

After adding the package dependency using the file menu "Add Package Dependencies" option, I am able to use the package on the iOS sim, but not on iOS devices or the macOS target.

The package.swift file appears to have definitions for 3 different versions of the library, "SwiftUIIntrospect", "SwiftUIIntrospect-Static", and "SwiftUIIntrospect-Dynamic".

**Comment by davdroman** [comment]
See https://github.com/siteline/swiftui-introspect/issues/318.

**Reply by DuncanMC** [reply]
@davdroman thanks for the link. You might want to add a comment about this in the docs, since projects created with the latest version of Xcode fail to build as soon as you add the introspect library to them.

**Reply by davdroman** [reply]
Good shout. I just added an illustrative screenshot to the README https://github.com/siteline/swiftui-introspect#swift-package-manager
